lib/pharos/phases/reset_host.rb

```diff
@@ -8,6 +8,10 @@ class ResetHost < Pharos::Phase
       def call
         logger.info { "Removing all traces of Kontena Pharos ..." }
         host_configurer.reset
+        if @config.addons.dig('kontena-lens', 'enabled')
```

Comment:
We probably should have `reset` in addon DSL. Does not make sense to hard-code addon reset logic here.